src/Abp.Zero.EntityFramework/EntityHistory/EntityHistoryHelper.cs

```diff
@@ -264,21 +264,19 @@ private ICollection<EntityPropertyChange> GetPropertyChanges(DbEntityEntry entit
                 var memberEntry = entityEntry.Member(propertyName);
                 if (!(memberEntry is DbPropertyEntry))
                 {
+                    // skipping other types of properties
```

Comment:
Let's begin comments with a capital letter. This make it easily distinguishable from code that is commented out.
```suggestion
                    // Skipping other types of properties
```

Reply:
👍🏻